### ISSUE TITLE
Staking MonyX coins for 52+ hours but No MinitReward

### ISSUE THREAD

**Issue by cryptovim84r** [monyxcoin/monyx-coin#182]
I have been staking MonyX coins for 52+ hours, now (with the QT Wallet running the entire time). But I have not yet received any "MinitReward". I've attached some images below.

Am I doing something wrong? Should all the coins be grouped together in one batch (i.e. 590 coins vs broken-up into 100 coin groupings) and/or etc...

![monyx_staking__2daysnoreward__01](https://user-images.githubusercontent.com/36458992/36175900-f18eacd6-10de-11e8-8b2f-2f6f8d8fa728.png)

![monyx_staking__2daysnoreward__02](https://user-images.githubusercontent.com/36458992/36175913-fab969b8-10de-11e8-816c-421fff56dbf7.png)

Thank you.

**Comment by chakramchirkut** [comment]
1.If you check on the right-hand corner, the mine axe icon, does it say x number of input? 
2.Secondly, do you have any bar showing how many open connections you have?
3.The mint reward comes 24hrs AFTER the  coinage has passed 1 day meaning, 1day after you see green color.
4. If it does not work after 24 hrs then go to help -> Debug-> console-> type repairwallet. This may help.
- Helping volunteerly.

**Comment by marcelvan** [comment]
Tutorial video would be awesome

**Comment by cryptovim84r** [comment]
@ chakramchirkut -  Thank you for your help. Answer below and more images

1.	When I hoover over the axes the message says “No suitable inputs were found”
![monyx_staking__2daysnoreward__03](https://user-images.githubusercontent.com/36458992/36181720-0c5ccf10-10f4-11e8-85da-83c2eb40893e.png)

2.	Regarding connections: For the past 2 days that messages fluctuates between 2 – 4 connections (and most often it is 2 or 3 connections but a few times it has been 4 connections.) 
![monyx_staking__2daysnoreward__04](https://user-images.githubusercontent.com/36458992/36181732-2065e8fc-10f4-11e8-8f71-d13dd4fad1e9.png)

3.	I don’t remember when the color turned green.. but if I understand you correctly the mint reward should occur pretty soon after 48 hours (correct?)… at this this point the coins have been in the desktop wallet with the wallet running for approximately 57 hours and counting
4.	So in your opinion should I try the { help -> Debug-> console-> type repairwallet.}
Or is there something else that is causing my issue?


Thank you.

**Comment by cryptovim84r** [comment]
Update – for possibly bug fix and others who may be affected

1. I decided to wait for a 3rd 24-hour period (Age column: = 3), to see if anything would change (it did not).
![monyx_staking__2daysnoreward__05__72hrs](https://user-images.githubusercontent.com/36458992/36214205-d907486a-1176-11e8-8339-dfee8333e72e.png)

2.  I then tried the “repairwallet”: { help -> Debug-> console-> type repairwallet.}. The returned message said "Passed"
![monyx_staking__2daysnoreward__05__repairpassed](https://user-images.githubusercontent.com/36458992/36214259-02b9ceee-1177-11e8-8bd3-74288c4d52df.png)

3. I then exited and restarted the program/wallet. 
	a. When it started up... while it was "syncing" I unlocked the wallet (again).
	b. After it synced-up... I moused over the Axes and Now I get a message saying "Stake miner is active 6 inputs being used for mining..."
![monyx_staking__2daysnoreward__06__restart](https://user-images.githubusercontent.com/36458992/36214683-1d1ff686-1178-11e8-896a-9f9f18fed108.png)


In Closing:
1. Is this some sort of bug?
2. Do I need to restart the program every 24 hours or after every "MintReward"?
3. Should/Can I just NOT run the wallet... and maybe only run/open once or twice per month and still receive the "MintReward"?

.

**Comment by Blitzkitten** [comment]
I found a couple of glitchy things like that first time I ran the wallet, but after closing and re-opening all was well.

option 3 is feasible! because your stakes gain weight over time you can open the wallet every so often, your chances of finding a block (and the reward given) will be higher.

I am unsure at what time stakes reach maximum probability but if you run your wallet once a week you'll be covered.

**Comment by monjil43** [comment]
Any new update @monyx...exchange is like dead...please do somthing..

**Comment by chakramchirkut** [comment]
@cryptovim84r 
Thanks mate for detailed screenshots.

So here is my observation as well.

1. It appears that somehow the "Stacke ready" (Green) inputs are not being cleared from cache i think, hence it requires restarting the wallet and re sync of the blockchain. This definitely needs to be looked at **Impact=Severity 3 Minor, Priority=Low, Category=bug)**

2. Now about the input :- as long as there is atleast 1 active input it means you are in the race to get some reward however as more miners/stackers have joined, you would have noticed the PoW difficulty has increased significantly..personally i have stopped keeping my pc on as i would rather buy from exchange right now at 15-16 cents. 
However if you are interested in desktop wallet stacking then i would advise to combine all the input by doing below.
When inputs are back to immature (purple).. then Go to Send coin -> and send all the coins (minus fees i think 0.00005 or something) to same  one address to yourself again.
this will make on single input in purple in the minting screen with more "weight" of the coin meaning all coins total together in one input. This will have better chances of getting reward.

3. You may have noticed that this coin has x11 algorithm meaning any mining rig or miner machine (personally owned) can be used to mine MYX which support x11 algorithm, now if you have one i suggest you use that otherwise the chances of getting reward are getting slimmer day by day.
General estimate during this week was those who have 16 K coins get about 8 coins after 30 hrs as reward.

4. The QT wallet is very generic and you can actually master it by learning more about it online, the code and source has not changed much since it was released for bitcoin long time ago.

5.There are advantages and disadvantages ot Locking / keeping unlocked wallet. it depends on strategy and coins etc...in your case specifically looking at your balance I guess it wont matter if you keep it locked and open it after couple of days or one a day OR even keep pc running 24/7  because the no of coins are not much. (comparing to the example 16k guy..above).

**Comment by revbukola** [comment]
Please how do I stake my monyx çoin? I have the coin in the monyx wallet I used to buy the coin during the ICO.

**Comment by chakramchirkut** [comment]
@revbukola - You leave them in your wallet,  make sure wallet is connected to the network and you have valid input.. check the responses in above list..with screenshots.

**Comment by revbukola** [comment]
I was able to buy just 99 monyx coins during the ICO and the coin are still there. Please is it safe? Then how do I move the 99coins to the exchange after it has been listed?

**Comment by cryptovim84r** [comment]
@ chakramchirkut

1. O.k. thanks… I’ll probably just try and start it up once a week or so…

2. And I came across advanced coin feature (to combine many inputs into one input) while researching more about staking wallets online

3. MonyX Coin price: Yeah it looks like now is “the time to buy” or “dollar-cost average” into a large position…. Hopefully after the 20th when MonyX’s Buy-Back program starts, and more info regarding the future direction of the platform (and all or most of the weak-hand/ impatient investors sale-off…) the price will start trending "northward" for a change.

------
4. I’m pretty new to crypto (and mining and staking) but I have indeed noticed the POW difficulty increasing.
I’ve also noticed the steady increase in “Network (GH/s)”: I assume that is the processing power that miner’s have devoted to the Monyx Blockchain. And I see also that the “network weight”-(which I assume is in some-way correlated to the number and age of coins being staked) has 4.5x since I posted my images.

5. I’ve recently gained some interest in mining… specifically some of these recent ICO coins. But I don’t have a mining rig and I’m not really looking to do any long-term mining… but just, perhaps, some short-term “speculative” mining on young coins (just after the ICO) while the difficulty is low.

I’ve done a little research on X11 mining and from my research, so far, it seems that it is possible to mine with a CPU, in the early stages of a coin’s life cycle and get O.K. results. One source said a Multi-core CPU with high memory works best. I have a few old PCs with CPUs’ that have 4 cores  (this source I believe hyper-threading didn’t matter, but one of them does have hyper-threading for 8-cores), one has 6mbL3 Cache and the other 8MB L3 Cache.

I’ve also seen that one can rent hash power and mine that way.

In your opinion has the POW difficulty for MonyX already passed what an average CPU can do? Or which option would you recommend. Also any software, sites, or information sources you could recommend would be appreciated. Thanks.

**Comment by Blitzkitten** [comment]
@cryptovim84r 

I wouldn't recommend mining on CPU.. the rewards for CPU mining are always pitiful. 
If you have a fairly modern GPU then you might be able to get some decent rewards. 

If you'd like to mine just for the sake of it, to give it a try out of interest then go for it, but don't expect decent rewards from CPU. 

I wouldn't rent hashing power off the bat either, in my opinion you'd be better off to just buy coins straight out. 

Basically, GPU mining will yield MUCH higher returns, if you don't have a decent one but want to give mining a go, use your CPU but don't expect it to be viable. 

Hope that helps

**Comment by cryptovim84r** [comment]
@Blitzkitten 

O.K. thanks for the info. I'll probably do some tests with the CPUs as well as a GPU (that I rely on for other functions) to see what the numbers look like first.


> @cryptovim84r

I wouldn't recommend mining on CPU.. the rewards for CPU mining are always pitiful.
If you have a fairly modern GPU then you might be able to get some decent rewards.

If you'd like to mine just for the sake of it, to give it a try out of interest then go for it, but don't expect decent rewards from CPU.

I wouldn't rent hashing power off the bat either, in my opinion you'd be better off to just buy coins straight out.

Basically, GPU mining will yield MUCH higher returns, if you don't have a decent one but want to give mining a go, use your CPU but don't expect it to be viable.

Hope that helps

Edit: What might be a good approach is to do a little mining with your CPU, measure how much reward you're getting for your hash rate.
Once you know your reward per hash you can then determine if renting hashing power is a viable option or not.

**Comment by chakramchirkut** [comment]
@Blitzkitten  Thanks mate for the inputs.
@cryptovim84r - I fully agree with above answer from Blitzkitten.

Honestly i was also looking to invest for mining but then dropped the idea because the amount of electricity consumed would be better off spending directly to buy coins cheaply. UNLESS you have 4-5 kwh solar with battery system to store as well which can be used at nigh time.

Other point you mentioned about early stage in new coins life, yes that is very true but I have noticed that with 8 core CPU i could not mine strayacoin, which was launched on 26 th Jan AUS day by one AUssie..it has already passed that point to mine with normal multicore laptop or desktop cpu so in general it better to buy cheaply at ICO stage.

BTW on ico, realy good one PolySwarm (competion to Augur (REP) is on 20 th Feb)

**Comment by cryptovim84r** [comment]
@chakramchirkut  Thanks for the info.
